src/System.Reflection.Metadata/src/System/Reflection/Metadata/Handles.cs

```diff
@@ -652,6 +652,18 @@ public override int GetHashCode()
         {
             return left._rowId != right._rowId;
         }
+
+        /// <summary>
+        /// Returns a handle to <see cref="MethodDebugInformation"/> corresponding to this handle.
+        /// </summary>
+        /// <remarks>
+        /// The resulting handle is only valid within the context of a <see cref="MetadataReader"/> open on the Portable PDB blob,
+        /// which in case of standalone PDB file is a different reader than the one containing this method definition.
+        /// </remarks>
+        public MethodDebugInformationHandle ToDebugInformationHandle()
```

Comment:
This is new since the API review. It is on MethodDefinitionHandle, it's little confusing in the github review, since all handles are in a single file. MethodDebugInformationHandle has the reverse translation method.